python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -57,6 +57,7 @@
 
 
 class RasterizeAlgorithm(QgisAlgorithm):
+
```

Comment:
Sorry, I fail to find the reference to this, can you point me towards it?
More important than how exactly, it's done: if possible we should also let autopep8 handle this.